README.md

```diff
@@ -129,9 +129,10 @@ ardrive upload-file --wallet-file /path/to/my/wallet.json --parent-folder-id "f0
         15. [Uploading Manifests](#uploading-manifests)
         16. [Hosting a Webpage with Manifest](#hosting-a-webpage-with-manifest)
         17. [Uploading With a Custom Content Type](#custom-content-type)
-        18. [Uploading a Custom Manifest](#custom-manifest)
-        19. [Uploading Files with Custom MetaData](#uploading-files-with-custom-metadata)
-        20. [Applying Unique Custom MetaData During Bulk Workflows](#applying-unique-custom-metadata-during-bulk-workflows)
+        18. [Uploading From a Remote URL](#remote-path)
```

Comment:
Why this choice of location in the README? I think it might make more sense to have this one somewhere between items 4 and 5 🤔 I'll read the actual copy and think more about it.

Reply:
Sure. I think we tried to build the ordering in a way that:
a) built upon concepts one after the other
b) optimized for most likely workflows sooner (as long as concepts from (a) weren't skipped)